src/libraries/System.Runtime.Experimental/src/System.Runtime.Experimental.csproj

```diff
@@ -0,0 +1,19 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Can we omit the src project entirely?  We should only need a reference in the package.

Reply:
Don't we need the type forward for the new interfaces (e.g. `System.INumber<>` and friends)?

Reply:
I was imagining that the shipping runtime assembly still contains those type-forwards.  Their presence in the runtime assembly should not offend any compilers 😉

Reply:
I agree with @ericsj. The shipping runtime assembly should have all the type forwarders necessary, or life will be difficult, as I really don't want to deal with an environment where we have a nuget package where the System.Runtime assembly is replaced by a standin (This will cause weird r2r issues in some cases). All we should need is an adjustment to the ref assembly, and to have the runtime assembly for System.Runtime contain the appropriate type forwarders.

Reply:
I believe this implies that you need to change the runtime assembly for System.Runtime to enable generic math which is a change not currently in this PR.

Reply:
Fixed by setting `ContractProject` as per our teams conversation.